modules/zivid/image.py

```diff
@@ -18,10 +18,10 @@ def __init__(self, impl):
         Raises:
             TypeError: If argument does not match the expected internal class.
         """
-        if not isinstance(impl, _zivid.ImageRGBA):
+        if not isinstance(impl, _zivid.ImageRGBA | _zivid.ImageBGRA):
```

Comment:
This syntax was introduced in Python 3.10. Unfortunately we still have to support older Python versions still, where this gives:
```
TypeError: unsupported operand type(s) for |: 'pybind11_type' and 'pybind11_type'
```

While we do test all supported Python versions in the CI, there is a hole in our test coverage: We do not test 2D capture stuff, because that does not work properly with file cameras. I suppose we can do some smoke-tests anyway, but I can deal with that.

For now you have to use the legacy syntax:
```
isinstance(impl, (_zivid.ImageRGBA, _zivid.ImageBGRA))
```